source/whats-new.txt

```diff
@@ -93,6 +93,14 @@ What's New in 4.0
      - ``BSONCXX_POLY_USE_SYSTEM_MNMLSTC``.
      - ``BSONCXX_POLY_USE_BOOST``.
 
+   - Drops support for the ``MONGOCXX_OVERRIDE_DEFAULT_INSTALL_PREFIX`` configuration
+     option. This option is implicitly set to ``OFF``.
+
+   - Sets the ``ENABLE_TESTS`` configuration option to ``OFF`` by default. To
```

Comment:
Question for tech reviewer: is this a breaking change?

Reply:
This is a build system breaking change which does not affect the API or ABI of the driver libraries. This only affects packaging recipes and/or build scripts which expect test targets to be enabled (most are likely already explicitly setting `ENABLE_TESTS=OFF`). It does not impact users of the driver libraries themselves.